python/ray/ray_perf.py

```diff
@@ -180,6 +201,30 @@ def multi_task():
 
     timeit("multi client tasks async", multi_task, n * m)
 
+    arr1 = np.ones((10, 1024, 1024))
+    arr2 = np.ones((10, 1024, 1024))
+
+    def data_intensive_task():
+        ray.get(large_arg_and_value.remote(arr1, arr2))
```

Comment:
Same comment here, I'm not sure if these microbenchmarks are useful (it's fine if we don't add any for data locality).